### ISSUE TITLE
Issue with Docker Image from Docker Registry

### ISSUE THREAD

**Issue by rafael** [kamon-io/docker-grafana-graphite#3]
Hi all, 

I was playing with this Docker Image and it is awesome. However I did notice an issue. When I use the image from the registry, I'm not being able to push the stats. However, when I build the container manually from the repo it works. Is it possible that the docker registry image is not up to date? 

Gracias !

**Comment by liaabi** [comment]
Hi, I am having the same issues. When running from the docker image and looking at the logs, it hangs after printing  INFO success: statsd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs).
I am trying to build the image from git now.

**Comment by liaabi** [comment]
When I build it from source I get the following errors:

Creating fuse device...
mknod: 'fuse-': Operation not permitted
makedev fuse c 10 229 root root 0660: failed
chown: cannot access '/dev/fuse': No such file or directory
dpkg: error processing package fuse (--configure):
 subprocess installed post-installation script returned error exit status 1
Processing triggers for libc-bin (2.19-0ubuntu6.5) ...
Errors were encountered while processing:
 fuse
E: Sub-process /usr/bin/dpkg returned an error code (1)
2015/01/15 11:42:02 The command [/bin/sh -c apt-get -y install  python-django-tagging python-simplejson python-memcache python-ldap python-cairo                            python-pysqlite2 python-support python-pip gunicorn supervisor nginx-light nodejs                            git wget curl openjdk-7-jre build-essential python-dev] returned a non-zero code: 100

**Comment by raboof** [comment]
I have no idea (and that's a separate issue I'd say :) )

I wonder if this could happen if you don't have fuse on the host machine? But perhaps I shouldn't make wild guesses in public :)

**Comment by liaabi** [comment]
Thanks, I opened a new issue.
I also have no idea :D

**Comment by ivantopo** [comment]
hey guys, I finally realized what was the problem when pulling the image from the registry: I naively assumed that when pushing a new image to the registry it will automatically become the `latest` image and anyone pulling `kamon/grafana_graphite` was going to get it but I was wrong... it is necessary to explicitly tag the image as `latest`. We recently upgraded the image to grafana 1.9.1 and correctly published it to the registry and after a couple tests if definitely works nice.

Let us know if you have any problems down the road, regards!
